index.d.ts

```diff
@@ -41,3 +41,22 @@ Run multiple promise-returning & async functions with limited concurrency.
 @returns A `limit` function.
 */
 export default function pLimit(concurrency: number): LimitFunction;
+
+export type LimitFunctionOption = {
```

Comment:
Plural

Reply:
Actually, just name it `Options` as I would like to make `pLimit` support an options-object too, for readability.